docs/fastdds/statistics/dds_layer/domainParticipant.rst

```diff
@@ -156,4 +156,7 @@ For the creation of an automatically enabled Datawriter, the priority for settin
     `GENERIC_STATISTICS_PROFILE`.
 
     The specific DataWriter profile defined in the `FASTRTPS_DEFAULT_PROFILES_FILE` XML needs to be named using the
-    statistic topic alias (see :ref:`statistics_topic_names` for the alias corresponding to each statistic topic).
+    same statistic topic alias or name (see :ref:`statistics_topic_names` for the alias corresponding to each statistic
+    topic) that has been used in the |DomainParticipantQos| |DomainParticipantQos::properties-api| ``fastdds.statistics``
```

Comment:
Maybe a reference to the [property](https://fast-dds.docs.eprosima.com/en/latest/fastdds/property_policies/non_consolidated_qos.html#statistics-module-settings) can be included, as it has been done for the environment variable.

Reply:
Done: f48fccbd Refs #15014: Reference to the property.